### ISSUE TITLE
Check links for units 1 and 2

### ISSUE THREAD

**Issue by Rek3650** [RITAccess/IECS#15]
The lessons in unit 1 and 2 need to have all of their links checked. They need to be visited to see if their still up and  then checked to see if screen readers can still read them. If they are down or unreadable then a suitable replacement should be found.

**Comment by Rek3650** [comment]
I am currently looking through unit 2

**Comment by baozi90** [comment]
Are we checking the entire site or just the page the link leads to?

**Comment by Rek3650** [comment]
just the page it links to.

**Comment by baozi90** [comment]
I'm going to start on Unit1 5-7.docx links.

**Comment by baozi90** [comment]
Regarding the Unit1 5-6.dox file: Are the blue ones, in screenshot, supposed to have links to it? Also, does it matter if wikipedia is repeated? 

<img width="409" alt="qq20150720-2 2x" src="https://cloud.githubusercontent.com/assets/11864006/8783415/ad09f568-2ee9-11e5-8a8e-17175fa2d749.png">

**Comment by retrogamer80s** [comment]
Hi,

The rubric is under the rubrics folder, and the website scavenger hunt is
in the Student Files folder for Unit 1.

They are docs not urls.

--Steph

**Comment by retrogamer80s** [comment]
It is ok if wikipedia is repeated.

--Steph

**Comment by baozi90** [comment]
Okay, thanks.

**Comment by baozi90** [comment]
About testing the urls, how to access a menu like this: 

https://archive.org/#maincontent
<img width="1275" alt="qq20150720-3 2x" src="https://cloud.githubusercontent.com/assets/11864006/8783821/381ce3c0-2eec-11e5-8e06-37bcaa1dd8f7.png">

**Comment by baozi90** [comment]
✓ VoiceOver: VO+A &/or VO+L and some tab
(tested in Safari)

【Unit1 5-7.dox】All links works
• http://zapatopi.net/treeoctopus/ ✓ 
• https://archive.org/#maincontent ✓ 
  https://archive.org ✓ 
  - overall VO+A and VO+L arrow keys works, except, not sure how to navigate the menu
• https://www.wikipedia.org ✓
• http://www.britannica.com ✓
• http://maps.yahoo.com/ ✓
• http://www.imdb.com ✓
• http://www.switchboard.com ✓
• http://www.yellowpages.com ✓
• http://www.howstuffworks.com ✓
• http://www.youtube.com ✓
• http://howcast.com ✓
  - While using VO-A, when inside the search textfield, a search pop-up appears and both of the follow happened at least once:
  1. when closed, VoiceOver starts from beginning of page again
  2. VoiceOver continues reading the rest of the page (ignoring the pop-up)
• http://www.delicious.com ✓
• http://www.stumbleupon.com ✓
• https://www.twitter.com ✓

**Comment by baozi90** [comment]
I'm pretty sure those are all the links in Unit 1.

Since it's only tested in Safari so far, Shannon (Chrome) and I (Firefox) will split up testing in Chrome and Firefox.

**Comment by baozi90** [comment]
✓ VoiceOver: VO+A and/or VO+L and some tab
(tested in Firefox)

【Unit1 5-7.docx】All links works
• http://zapatopi.net/treeoctopus/ ✓ 
• https://archive.org ✓
  - overall VO+A and VO+L arrow keys works, except, not sure how to navigate the menu
  - skips over the "SEE MORE" link:
<img width="271" alt="seemore" src="https://cloud.githubusercontent.com/assets/11864006/8809110/7cc824ee-2fb4-11e5-8920-4a223d782596.png">


• https://www.wikipedia.org 
• http://www.britannica.com 
• http://maps.yahoo.com/ 
• http://www.imdb.com 
• http://www.switchboard.com 
• http://www.yellowpages.com 
• http://www.howstuffworks.com 
• http://www.youtube.com 
• http://howcast.com 
  - While using VO-A, when inside the search textfield, a search pop-up appears and both of the follow happened at least once:
      1. when closed, VoiceOver starts from beginning of page again
      2. VoiceOver continues reading the rest of the page (ignoring the pop-up)

• http://www.delicious.com 
• http://www.stumbleupon.com 
• https://www.twitter.com 

【Unit1 11-14 iSonic alternative.docx】
• http://www.cs.umd.edu/hcil/audiomap/ 

【Unit1 15-16 docx】
• http://www.justriddlesandmore.com/direct.html 

【Unit1 17-19.docx】
• http://www.csunplugged.com

**Comment by smcintosh881** [comment]
✓ VoiceOver: VO+ arrow keys
(Chrome)
【Unit1 5-7.docx】All links works
• http://zapatopi.net/treeoctopus/ ✓ 
• https://archive.org ✓ (This website does some wonky stuff with the screen reader. You can access everything, but the labels are strange)
![screen shot 2015-07-21 at 4 47 50 pm](https://cloud.githubusercontent.com/assets/9065471/8812138/53b91644-2fc8-11e5-9a83-20cc35ba210a.png)
This is what VoiceOver shows when on the exit button. It reads out "link row-boat"
Chrome only worked with VO+arrow keys

• https://www.wikipedia.org 
• http://www.britannica.com 
• http://maps.yahoo.com/ 
• http://www.imdb.com 
• http://www.howstuffworks.com 
• http://www.youtube.com 
• http://www.delicious.com 
• http://www.stumbleupon.com 
• https://www.twitter.com

• http://www.switchboard.com 
• http://www.yellowpages.com 
         Replace switchboard.com and yellowpages.com with http://www.whitepages.com/ 
         (works well with Chrome, Safari and FireFox)
        
• http://howcast.com
       •While using VO-A, when inside the search textfield, a search pop-up appears and both of the follow happened at least once:
       •when closed, VoiceOver starts from beginning of page again VoiceOver continues reading the rest of the page (ignoring the pop-up)
       •Not overly accessible. Seems like it could be difficult because of the pop-up

【Unit1 11-14 iSonic alternative.docx】
• http://www.cs.umd.edu/hcil/audiomap/
       • Really good one. Has great description of the images. 

【Unit1 15-16 docx】
• http://www.justriddlesandmore.com/direct.html
         • Some of the areas VoiceOver reads are very long

【Unit1 17-19.docx】
• http://www.csunplugged.com
          • This is not a great one. The images are not properly labeled, however the it is possible to get to the file that is needed to be downloaded without obstacle. 
          • PDF is opened in web so if they want to save they will need to use "Command" + "S"

**Comment by smcintosh881** [comment]
I cant really find anything similar to http://archive.org 
If we have them start at http://archive.org/web/ which is a less busy page. The labels are still poor for navigation. 
The different versions of the website are uploaded as images, which don't have labels explaining the image, therefore it will be difficult/impossible for students to know any difference between a new and old version of a website. 
I think we might need to replace this activity in the lesson plan. Currently it is used to compare the schools current website with an older version of it. I don't think this activity would be best for all students.

**Comment by baozi90** [comment]
There are more than a couple sites that seems to skip over the navigation like YouTube with VoiceOver on Firefox.
